src/containers/ActivityContainer/FilterModal.tsx

```diff
@@ -0,0 +1,254 @@
+import React, { useEffect, useRef, useState } from 'react';
+import Image from 'next/image';
+
+import CButton from 'src/components/CButton';
+import { useAppSelector } from 'src/hooks/useRedux';
+import { IToken } from 'src/reducers/tokens';
+
+import close from 'public/images/whiteClose.svg';
+import blackClose from 'public/images/close.svg';
+import searchLogo from 'public/images/search.svg';
+import { IFilterTokens } from 'src/constants/types';
+import defaultToken from 'public/images/defaultToken.svg';
+import CBottomSheet from 'src/components/CBottomSheet';
+
+type ModalProps = {
+  open: boolean;
+  closeModal: () => void;
+  initialSentChecked: boolean;
+  selectedTokenValue: IToken[];
+  setIsOpen: (_: boolean) => void;
+  initialReceivedChecked: boolean;
+  setSelectedTokenValue: React.Dispatch<React.SetStateAction<IToken[]>>;
+  handleSubmitFilter: (filteredValuesObject: IFilterTokens, selectedTokens: IToken[]) => void;
+};
+
+const FilterModal = ({
+  open,
+  setIsOpen,
+  closeModal,
+  handleSubmitFilter,
+  selectedTokenValue,
+  setSelectedTokenValue,
+  initialReceivedChecked,
+  initialSentChecked,
```

Comment:
You should try to give more meaning full names to these props (both here and the place they're declared).

Reply:
What is not meaningful to you here? can you explain?